packages/paste-website/src/component-examples/ChatLogExamples.ts

```diff
@@ -0,0 +1,297 @@
+        <ChatEvent>
+          <strong>Gibby Radki</strong> has joined the chat・3:34 PM
+        </ChatEvent>
+        <ChatMessage variant='inbound'>
+          <ChatBubble>
+            Hello, what can I help you with?
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by Gibby Radki at 3:35 PM">
+            <ChatMessageMetaItem>
+              <Avatar name="Gibby Radki" size="sizeIcon20" />
+              Gibby Radki
+            </ChatMessageMetaItem>
+            <ChatMessageMetaItem>3:35 PM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatMessage variant='outbound'>
+          <ChatBubble>
+            Hi! What is your return policy?
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by you at 3:35 PM">
+            <ChatMessageMetaItem>3:35 PM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatMessage variant='inbound'>
+          <ChatBubble>
+            You have 30 days to send your items back. Here is the full policy if you would like to read more.
+          </ChatBubble>
+          <ChatBubble>
+            <ChatAttachment attachmentIcon={<DownloadIcon color="colorTextIcon" decorative />}>
+              <ChatAttachmentLink href="#">RETURN_POLICY.pdf</ChatAttachmentLink>
+              <ChatAttachmentDescription>17 MB</ChatAttachmentDescription>
+            </ChatAttachment>
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by Gibby Radki at 3:37 PM">
+            <ChatMessageMetaItem>
+              <Avatar name="Gibby Radki" size="sizeIcon20" />
+              Gibby Radki
+            </ChatMessageMetaItem>
+            <ChatMessageMetaItem>3:37 PM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatMessage variant='outbound'>
+          <ChatBubble>
+            Thank you
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by you at 3:40 PM">
+            <ChatMessageMetaItem>3:40 PM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatBookend>
+          <ChatBookendItem>Today</ChatBookendItem>
+        </ChatBookend>
+        <ChatMessage variant='outbound'>
+          <ChatBubble>
+            I have shipped the item back, how long will it take to get the refund?
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by you at 11:27 AM">
+            <ChatMessageMetaItem>11:27 AM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+          <ChatMessageMeta aria-label="(read)">
+            <ChatMessageMetaItem>Read</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatMessage variant='inbound'>
+          <ChatBubble>
+            You should recieve the refund within 10 days.
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by Gibby Radki at 11:29 AM">
+            <ChatMessageMetaItem>
+              Gibby Radki
```

Comment:
Todo: the last two mentions of Gibby need to have an avatar included to match the first part of the the example.